src/components/portfolioPage/Portfolio.jsx

```diff
@@ -37,22 +37,23 @@ class Portfolio extends Component {
 
   render() {
     let viewpoints = this._getViewpoints();
-    let corpora = this._getCorpora();
     let attributes = new Items(this.state.items)
       .getAttributes()
       .map(([key, value]) => key.concat(' : ', value))
       .map(x => ({[x]: {name: x}}));
     let candidates = this.state.viewpoints.concat(attributes);
     const urlParams = new URLSearchParams(window.location.search);
     const selectionJSON = JSON.parse(urlParams.get('t'));
+    // Normal items have numbers in their names, but building maps don't,
+    // so the regex only matches building maps.
     const visitPossible = this.state.visitMap
       && selectionJSON
       && selectionJSON.type === 'intersection'
-      && selectionJSON.data.length === 1
```

Comment:
I will add this line again since its deletion make browsing fail (reproduction scenario: select a topic and then unselect it).

Reply:
Hmm. But deleting this line seems necessary for your feature... So I suppose your feature needs a little more work...

Reply:
Maybe `>0 `?